modules/gltf/editor/editor_scene_importer_gltf.cpp

```diff
@@ -29,13 +29,13 @@
 /*************************************************************************/
 
 #ifdef TOOLS_ENABLED
-
```

Comment:
Please keep separation lines like this, it's consistent with the rest of the codebase and more readable.